modules/Components/src/client/BasicFileValidateAndSaveView.html

```diff
@@ -1,17 +1,17 @@
 <script type="text/template" id="BasicFileValidateAndSaveView">
     <div class='bv_outerContainer container' >
-        <div class="bv_moduleTitle row-fluid span7" style="padding-bottom: 0px;margin-bottom: 0px;">Generic Data Parser</div>
-        <div class="row-fluid span7 bv_additionalValuesForm"></div>
-        <div class="bv_resultStatus row-fluid span7"></div>
+        <div class="bv_moduleTitle row-fluid span11" style="padding-bottom: 0px;margin-bottom: 0px;">Generic Data Parser</div>
+        <div class="row-fluid span11 bv_additionalValuesForm"></div>
+        <div class="bv_resultStatus row-fluid span11"></div>
         <div class='bv_fileUploadWrapper row-fluid span12'>
-            <div class='well span7 bv_fileUploadDirections'>
+            <div class='well span11 bv_fileUploadDirections'>
                 To upload the <b>required data file</b>, click the "Browse Files…" button and select a file.
             </div>
             <div class='bv_parseFile'></div>
-            <div class="bv_reportFileFeature well span7" style="background:white;padding-bottom:0;margin-left:0;">
+            <div class="bv_reportFileFeature well span11" style="background:white;padding-bottom:0;margin-left:0;">
                 <div class="control-group bv_group_attachReportFile">
-                    <input type="checkbox" name="bv_attachReportFile" class="bv_attachReportFile" style="margin-top:0px;margin-bottom:3px;"/>
-                    <label class="control-label bv_attachReportCheckboxText pull-right" style="width:200px;margin-bottom:20px;margin-right:410px;">Attach optional report file
+                    <label class="control-label checkbox" style="width:200px;padding-left:20px;margin-bottom:20px;">
+                        <input type="checkbox" name="bv_attachReportFile" class="bv_attachReportFile"/>Attach optional report file
```

Comment:
When expanded to span 11, this didn't look right but the images attachment feature looked correct, I matched the styling to the images attachment and verified this still works.